packages/docs-page/server/index.ts

```diff
@@ -117,7 +117,7 @@ export function generateStaticPaths({
   const loader = new FileSystemLoader({
     navDataFile,
     localContentDir,
-    product: product?.slug,
+    product: product?.slug ?? '',
```

Comment:
```suggestion
    product: product.slug,
```

Reply:
And, I think [product isn't supposed to be optional](https://github.com/hashicorp/react-components/pull/481/files#diff-99829c81bc27331948908774fd93aa5a7e75b42d97274484f95d2c11d133025fR102) 🤔 

@BRKalow?

Reply:
Right, it probably shouldn't be optional as we need it for the edit this page link.

Reply:
👍